docs/release-notes/4257-which.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Fixes**
+
+-  Support running in containers which do not have ``/bin/which``, like python-slim. The error was
```

Comment:
suggest:
"Supports running in containers which do not have a ``/bin/which`` path, such as python-slim. The error was caused by accidentally ..."